src/components/accounts/RecoverAccount.js

```diff
@@ -0,0 +1,89 @@
+import React, { Component } from 'react'
+import { connect } from 'react-redux'
+import { isValidPhoneNumber } from 'react-phone-number-input'
+
+import { Wallet } from '../../utils/wallet'
+
+import RecoverAccountSection from './RecoverAccountSection'
+import RecoverAccountForm from './RecoverAccountForm'
+import RecoverAccountContainer from './RecoverAccountContainer'
+import { requestCode, validateCode } from '../../actions/account';
+
+class RecoverAccount extends Component {
+   state = {
+      loader: false,
+      phoneNumber: '',
+      isLegit: false,
+   }
+
+   componentDidMount = () => {
+      this.wallet = new Wallet()
+   }
+
+   handleChange = (e, { name, value }) => {
+      this.setState(() => ({
+         [name]: value,
+         isLegit: this.isLegitField(name, value)
+      }))
+   }
+
+   isLegitField(name, value) {
+      // TODO: Use some validation framework?
```

Comment:
https://www.npmjs.com/package/react-validation

Reply:
looks pretty cool, thanks!

Reply:
let's not use "isLegitField". It's not common language for this. https://www.merriam-webster.com/thesaurus/leg%20(it)
the word to use here is "valid"

Reply:
you are linking verb definition, which has no relation to "legit" adjective:

<img width="301" alt="Screen Shot 2019-05-16 at 12 43 27 PM" src="https://user-images.githubusercontent.com/126710/57882301-3ce82980-77d8-11e9-8fcb-57dd48fc94d1.png">